src/app.md

```diff
@@ -271,6 +272,54 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-file-skip-dot] #ex-file-skip-dot
```

Comment:
Rendering and link gets broken due to missing colon
```diff
@@ -272,7 +272,7 @@ fn run() -> Result<()> {
 # quick_main!(run);

-[ex-file-skip-dot] #ex-file-skip-dot
+[ex-file-skip-dot]: #ex-file-skip-dot
 <a name="ex-file-skip-dot"></a>
 ##  Traverse directories while skipping dotfiles
```

you can check rendering with mdbook
```bash
cargo install mdbook
cd ./path/to/rust-cookbook
mdbook serve -o
```